### DISCUSSION TITLE
Uber Duck rate limiting with paid sub?

### DISCUSSION THREAD

**Discussion by Beebopcat** [karashiiro/TextToTalk#158] [General]
Hey,

This is my favorite plugin! I do hope that Uberduck will be hammered out to perfection though. I understand that there is no rate limiting with a paid sub, any tier, of Uberduck. I paid for their $10/month sub, and there is what appears to be a delayed start between the text to dialogue speaking. Hands down, Uberduck has the most amazing voices, which would turn this game's dialogue into an entirely different experience. Am I doing something wrong, or is the delay in Text to Speech supposed to happen? Thank you!

**Comment by karashiiro** [comment]
The delay depends on how long it takes to generate the voices, and on your latency to Uberduck's servers. If I remember correctly, Uberduck had a tool on their website to preview voices - there should be a similar delay if you use that.

**Reply by Beebopcat** [reply]
Thank you for the response. You are a legend! Also, anything better than Uberduck? I couldn't do the other voice services aside from Amazon Polly. Thank you!

**Reply by karashiiro** [reply]
At the moment, I think Polly might be as good as or better than Uberduck, but with a more limited voice selection. I have ElevenLabs support pending as well, and apparently they're even better, but that work isn't done, yet.

**Reply by Beebopcat** [reply]
OK. Well, truly, thank you for your contribution to the community of FFXIV. I look forward to your future works.
